concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-365	365	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Bruder des eigenen Mannes	brother of one's husband		
+Steinthal-1875-1565-366	366	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Mann der eigenen Schwester	husband of one's sister		
+Steinthal-1875-1565-367	367	Schwager, Schwägerin (meiner Frau Bruder, meiner Schwester Mann; meines Bruders Frau; nennt eine Frau ihren Schwager und ihre Schwägerin ebenso? also ihres Mannes Bruder, ihrer Schwester Mann, ihres Bruders Frau)	Frau des eigenen Bruders	wife of one's brother		
+Steinthal-1875-1565-368	368	Greis	Greis	old man	406	OLD MAN
+Steinthal-1875-1565-369	369	alte Frau	alte Frau	old woman	907	OLD WOMAN
+Steinthal-1875-1565-370	370	Er ist alt.	Er ist alt.	He is old.		
+Steinthal-1875-1565-371	371	Er altert.	Er altert.	He is ageing.		
+Steinthal-1875-1565-372	372	Er bekommt graue Haare.	Er bekommt graue Haare.	He is greying.		
+Steinthal-1875-1565-373	373	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist gestorben.	He died.		
+Steinthal-1875-1565-374	374	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist todt.	He is dead.		
+Steinthal-1875-1565-375	375	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist alt geworden.	He grew old.		
+Steinthal-1875-1565-376	376	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist nicht alt geworden.	He died at a young age.		
+Steinthal-1875-1565-377	377	Er ist gestorben, todt, alt geworden, nicht alt geworden, noch jung.	Er ist noch jung.	He is still young.		
+Steinthal-1875-1565-378	378	Waise	Waise	orphan	1149	ORPHAN
+Steinthal-1875-1565-379	379	Familie	Familie	family	194	FAMILY
+Steinthal-1875-1565-380	380	Stamm	Stamm	tribe	1549	TRIBE
+Steinthal-1875-1565-381	381	Volk	Volk	people	789	PEOPLE
+Steinthal-1875-1565-382	382	Leib (Körper)	Leib (Körper)	body	1480	BODY
+Steinthal-1875-1565-383	383	Leichnam	Leichnam	dead body	767	CORPSE
+Steinthal-1875-1565-384	384	Knochen	Knochen	bones	1394	BONE
+Steinthal-1875-1565-385	385	Skelett	Skelett	skeleton	2668	SKELETON
+Steinthal-1875-1565-386	386	Mark	Mark	bone marrow	1835	MARROW
+Steinthal-1875-1565-387	387	Fleisch	Fleisch	flesh	2259	FLESH
+Steinthal-1875-1565-388	388	Blut	Blut	blood	946	BLOOD
+Steinthal-1875-1565-389	389	Adern	Adern	veins	1924	VEIN
+Steinthal-1875-1565-390	390	Haut	Haut	skin	763	SKIN
+Steinthal-1875-1565-391	391	Haar	Haar	hair	2648	HAIR (HEAD)
```

Comment:
Is there an indication that it refers exclusively to the head hair? If not, I would map it to more general HAIR.